### ISSUE TITLE
Añadir un campo de texto libre al perfil de usuario

### ISSUE THREAD

**Issue by deivid-rodriguez** [alabs/nolotiro.org#620]
Aquí las usuarias podrán indicar todo aquello que repitan una y otra vez regalo tras regalo.

* Zona.
* Horarios de entrega.
* Reglas/avisos sobre los regalos.
* Etc.

**Comment by abelkuest** [comment]
al rellenar dicho perfil podría añadir justo lo que pones para que a la gente le quede claro lo que puede poner

**Comment by deivid-rodriguez** [comment]
Sí. Hay que ver cómo presentarlo visualmente para que se entienda bien.
